### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by MasumaJaffery**
## Changes:-
- JS linters add
- Popup window insert in JS file 
- CSS file updated

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -277,21 +305,58 @@ body {
...
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 1.6rem;
+  padding-bottom: 2.4rem;
+  margin: auto;
+  z-index: 5;
+  box-shadow: 0 0 20px rgb(84, 84, 84);
+}
```
Comment (taldr27) [inline style.css:324]: Awesome work on the desktop version of your popup! I have a couple of friendly suggestions to make it even more professional and aligned with the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=Gl1dsCnjASkVGKBM-0):

- [ ] Please review the space between your image and the bottom of the popup, as **it appears slightly different from the template**. 🤓
- [ ] I noticed that the placement of your buttons in the desktop version doesn't match the template. To achieve a closer resemblance, kindly position the **right button in the correct place**. 🙌

  ### Your design:
  image

  ### Figma design:
  image

  **_Just a friendly reminder that adhering to the template is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Thanks for considering these suggestions, and keep up the great work!_**
image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello team, @MasumaJaffery, @Alibaba2023 , 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional and customized README. 
✅ Linter tests passing.
✅ Correct use of GitHub Flow.
✅ Populating the popup dynamically.
✅ Popup functionality working fine.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: style.css [hunk 2]
```diff
@@ -277,21 +305,58 @@ body {
...
```
Comment (mahbubul14) [inline style.css:324]: - [ ] As the previous reviewer suggested, Kindly try to match your popup design to the Figma design. You can reduce the space between your image and the bottom of the popup.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
pop-figma | pop_fig

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @Team,**

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Popup functionality working properly.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 2 above)
Comment (Alibaba2023) [reply style.css:324]: Hi  @mahbubul14  ;
## 
Screenshot 2023-05-19 071342
But the popup menu on my computer is like this:-

Comment (Alibaba2023) [reply style.css:324]: @mahbubul14 
Can you check the project again?
On my computer, the popup is working well.

**Review by divinecharlotte — APPROVED** [review]
# Hello @Alibaba2023     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- Linters are passing 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**